### DISCUSSION TITLE
How to mark items as fixed or closed ?

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#51] [Fixed Issues]
Is there some way to mark these Discussions as fixed, closed or answered ? Rern seems to have disabled 'Issues' on this board which does have the ability to make them as fixed or closed.

Chris

**Comment by cYCL157** [comment]
good question -- i looked for one that I opened and thought was closed.  not trying to add to rern's workload, but issues seem like they would be better way to manage this.  if there is a suggestion on how we could help, i am sure one or more of us would be willing.

**Comment by ceejayemm** [comment]
It appears that there is a new discussions category available called 'Fixed Issues'.  Click on the Cog to the right of the topic header and select 'Fixed Issues' to close a topic.

Chris

**Reply by cYCL157** [reply]
cool. i can see that I can do this for ones that i open, which i guess makes sense. it's interesting that this issue itself still shows as unanswered, despite being marked as fixed and having..........responses. Is there a way to "accept" an answer @ceejayemm ?
